opencanary/logger.py

```diff
@@ -99,7 +99,8 @@ class LoggerBase(object):
 
     def sanitizeLog(self, logdata):
         logdata['node_id'] = self.node_id
-        logdata['local_time'] = datetime.utcnow().strftime("%Y-%m-%d %H:%M:%S.%f")
```

Comment:
we will need to keep this line in however because some current users and systems are:
a) expecting a `local_time` variable
b) have adjusted to what it has been i.e. `datetime.utcnow().strftime("%Y-%m-%d %H:%M:%S.%f")`

So we are essentially just adding logdata that will have better definitions, but not removing or changing existing logdata fields incase.

Reply:
Is it worth putting a note in the log that this local_time is deprecated and will be removed in a future version? I just think it is a odd having a log entry of local time when it isn't (unless you live in a couple of countries in Europe!)

Reply:
Ye i think that works. It definitely something we will try move away from but not just yet

Reply:
thanks so much for the back and for btw. Its helpful!